src/main/resources/interface-specification.yml

```diff
@@ -6751,6 +6751,103 @@ paths:
     parameters:
       - $ref: '#/components/parameters/CorrelationIdHeader'
       - $ref: '#/components/parameters/IpAddress'
+    get:
+      tags:
+        - clients
+      summary: Returns a key by client and key identifier (kid).
+      description: 'Given a client and key identifiers it returns the corresponding key, if any'
+      operationId: getClientKeyById
+      parameters:
+        - name: clientId
+          in: path
+          description: ID of the client to look up
+          required: true
+          schema:
+            type: string
+            format: uuid
+        - name: keyId
+          in: path
+          description: the unique identifier of the key (kid) to lookup
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: returns the corresponding key
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PublicKey'
```

Comment:
I think this endpoint is used to return the actual value of the public key, but `PublicKey` model does not contain this information

Reply:
I probably mistook this endpoint for `/clients/{clientId}/encoded/keys/{keyId}`.
Let's wait for frontend confirmation and then on my side we can proceed

Reply:
Further investigations are required in a next refinement